deploy/aws/java11Exec/src/main/java/poseidon/SimpleMakefile.java

```diff
@@ -21,7 +22,7 @@ class InvalidMakefileException extends Exception {}
 class SimpleMakefile {
 
     // This pattern validates if a command is a make command.
-    private static final Pattern isMakeCommand = Pattern.compile("^make(?:\\s+(?<startRule>\\w*))?$");
+    private static final Pattern isMakeCommand = Pattern.compile("^make(?:\\s+(?<startRule>\\w*))?(?<assignments>(?:.*?=\\\".*?\\\")+)?$");
```

Comment:
I am not sure whether the `"` are required (or optional), but otherwise this MR looks fine. Thanks again ;)

Reply:
```suggestion
    private static final Pattern isMakeCommand = Pattern.compile("^make(?:\\s+(?<startRule>\\w*))?(?<assignments>(?:.*?=.*?)+)?$");
```

Reply:
If we remove them, do we need to adjust the slicing here as well?

https://github.com/openHPI/poseidon/blob/a733daffc8644fe83e389c2c25bfcd11e63dab9c/deploy/aws/java11Exec/src/main/java/poseidon/SimpleMakefile.java#L94

Reply:
Good catch, thanks